src/ln/msgs.rs

```diff
@@ -472,6 +473,13 @@ pub struct CommitmentUpdate {
 	pub(crate) commitment_signed: CommitmentSigned,
 }
 
+
+/// Struct used to return spendable_outputs from ChannelMonitor to KeysInterface
+pub struct SpendableOutputs {
```

Comment:
Why create a struct for this instead of just putting the SecretKey/Script/OutPoint in the event/API directly?